atomistics/calculators/lammps/libcalculator.py

```diff
@@ -436,6 +436,7 @@ def calc_molecular_dynamics_langevin_with_lammpslib(
             Tstart=Tstart,
             Tstop=Tstop,
             Tdamp=Tdamp,
+            seed=seed,
```

Comment:
I though the seed is only used in `LAMMPS_VELOCITY`, as the `LAMMPS_VELOCITY` is not included in the `else` part it should not be necessary here. Or am I missing something?